### DISCUSSION TITLE
Support camera?

### DISCUSSION THREAD

**Discussion by borgqueenx** [Tasshack/dreame-vacuum#486] [Ideas]
I don't know if it can happen, but it would be epic if the camera can be supported to live watch (and record?) the livestream the dreame vacuum cleaners have.

This way, we can use the devices to send them around the house and check up on certain things. For example: Motion sensor detected a person> send dreame vacuum cleaner to investigate.

**Comment by Tasshack** [comment]
Dreame uses two different professional streaming services with their own players for the camera feature (depending on the region) so it would require insane amount of reverse engineering to make it working with the HA. It is not impossible but I don't have much free time to do something like this.
Actually this is the main reason that Mi Home app does not have camera streaming feature because you cannot put a native android/ios streaming library in a Mi Home plugin.
